lib/package.rb

```diff
@@ -87,11 +88,11 @@ def self.get_deps_list(pkgName = name, hash: false, include_build_deps: 'auto',
        !exclude_buildessential && \
        !@checked_list.keys.include?('buildessential')
 
-      deps = { 'buildessential' => [:build] }.merge(deps)
+      deps = { 'buildessential' => [[:build]] }.merge(deps)
     end
 
     # parse dependencies recursively
-    expandedDeps = deps.uniq.map do |dep, depTags|
+    expandedDeps = deps.uniq.map do |dep, (depTags, ver_check)|
```

Comment:
We should decide if we're using camelCase or not and be consistent.

Reply:
Change made